### ISSUE TITLE
Rotational displacement

### ISSUE THREAD

**Issue by ericmazur** [ericmazur/PnPbook#33]
From Toby Dittrich: We have discovered a mistake in all introductory physics texts including yours recently published. I hope you carefully review this submission and please do comment on it, as your opinion is extremely valuable to us. Hopefully, you will agree with our discussion. I have presented this paper to many at the University of Colorado Boulder, including Dr. Steven Pollock (CU is my alma matter). They agreed with this approach. At the Pacific Northwest Association for College Physics (I am also PNACP Executive Officer) conference last month my lecture was very well received as well.

**Comment by ericmazur** [comment]
![toby](https://cloud.githubusercontent.com/assets/2316175/9578116/55875ea6-4fb5-11e5-82f1-8c835efb7c70.jpg)

**Comment by JohnDenker** [comment]
The paper attached in the previous comment offers a syllogism:
- Major premise:  angular position and angular velocity are the same kind of thing.
- Minor premise:  angular velocity is a vector.
- Conclusion: therefore angular position is a vector, too.

I am 100% happy with the major premise, but I reject the minor premise and the conclusion.

We will discuss the pedagogical angle in a moment, but first let's discuss the fundamental principles.  There is exceedingly strong evidence that angles and angular velocities (among other things) are well represented as **_bivectors**_.  The value of the bivector approach over competing approaches becomes particularly clear as soon as you try to construct a unified view of rotations in 2D, 3D, and 4D.  The value becomes even more stunning when you apply the formalism to other things, such as electromagnetism.

>   In the special case of three dimensions, you can choose an arbitrary basis and then use the Hodge dual to "associate" each bivector with a pseudovector ... but even then it's a pseudovector (not really a vector) ... and the pseudovector idea fails miserably in 2D, 4D et cetera.

Rather than making excuses as to why the Rodregues vectors don't add in a nice way, I would much rather just add the bivectors.  Works fine.

You can add bivectors geometrically (edge-to-edge), just as you would add vectors geometrically (tip to tail).

Pedagogical remark:  Suppose the goal is to explain gyroscopic precession.  I can teach somebody about bivectors and then explain precession in less time than it would take to explain precession alone, even under the highly dubious assumption that the student already understood cross products.  I hold up cardboard models of three bivectors:  A + B = C.  Easy peasy.

Bivectors give us a unified view of:
- complex numbers
- quaternions
- rotations in 2D
- rotations in 3D
- rotations in 4D
  - including boosts
- the electromagnetic field in 2+1D
- the electromagnetic field in 3+1D
- all without the slightest need of a right-hand rule

Real-world applications include
- autopilots
- flight simulators
- robot arms
- computer graphics systems
- anything else that requires maneuvering in 3D

If you are working on such a system, people would laugh at you if you didn't use bivectors (which are the same thing as quaternions, if we restrict attention to 3D).

Bottom line:  You can represent a 3D rotation using a Rodrigues vector if you really want to, but it is by no means the best representation, much less the only representation.
